README.md

```diff
@@ -94,7 +94,7 @@ Before contributing, please review the [Contribution Flow](https://github.com/me
   ```bash
   $ make site
   ```
-  _Note: From the Makefile, this command is actually running `$ bundle exec jekyll serve --drafts --livereload`_
```

Comment:
Given that this italicized note intends to convey a small insight to the reader, use of the word "actually" is befitting in that the assumption is that the reader didn't _actually_ realize what was happening behind the scenes. I suggest leaving as-is, unless you have separate justification...